### ISSUE TITLE
core.Model object doesn't take pretrained argument 

### ISSUE THREAD

**Issue by apar-t** [alankbi/detecto#83]
**Describe the bug**
class detecto.core.Model(classes=None, device=None, pretrained=True)

**Code and Data**
Share the code that caused the error as well as an example image/label from your dataset.

**Environment:**
 - OS: 
 - Python version: 
 - Detecto version:
 - torch version: 
 - torchvision version: 

**Additional context**

**Comment by alankbi** [comment]
Be sure you're using the latest version: https://github.com/alankbi/detecto/releases/tag/v1.2.1

**Comment by bgriffen** [comment]
Thanks @alankbi - awesome package. How do I stop downloading all of the models? Running on GKE.

```
Downloading: "https://download.pytorch.org/models/resnet50-0676ba61.pth" to /root/.cache/torch/hub/checkpoints/resnet50-0676ba61.pth
Downloading: "https://download.pytorch.org/models/fasterrcnn_resnet50_fpn_coco-258fb6c6.pth" 
``` 

I tried:

```python
from detecto.core import Model
model = Model(pretrained=False).load(model_full_path,["a","b","c"])
```

but it still keeps downloading them on the cluster when pipeline gets to the detecto step. I am using a trained model of my own given by `model_full_path`.
